conjur-api.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency 'activesupport'
```

Comment:
Why is that? The only requirement currently in the gemspec is >= 1.9, which 2.2.2 certainly meets. Presumably, activesupport's gemspec specifies whatever requirement it has.